src/components/mods/modsTable.tsx

```diff
@@ -370,28 +458,52 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
 
 
             if (
-                mapCountRange[0] !== undefined ||
-                mapCountRange[1] !== undefined
+                selectedModTypes.length &&
+                !selectedModTypes.includes(modWithInfo.type)
+            ) {
+                return false;
+            }
+
+
+            if (
+                debouncedPublisherQuery &&
+                !modWithInfo.publisherName.toLowerCase().includes(debouncedPublisherQuery.trim().toLowerCase())
+            ) {
+                return false;
+            }
+
+
+            if (
+                publicationDateRange[0] !== undefined ||
+                publicationDateRange[1] !== undefined
             ) {
                 if (
-                    mapCountRange[0] !== undefined &&
-                    modWithInfo.Map.length < mapCountRange[0]
+                    publicationDateRange[0] !== undefined &&
+                    modWithInfo.timeCreatedGamebanana < publicationDateRange[0]
                 ) {
                     return false;
                 }
 
                 if (
-                    mapCountRange[1] !== undefined &&
-                    modWithInfo.Map.length > mapCountRange[1]
+                    publicationDateRange[1] !== undefined &&
+                    modWithInfo.timeCreatedGamebanana > publicationDateRange[1]
                 ) {
                     return false;
                 }
             }
 
 
             if (
-                selectedModTypes.length &&
-                !selectedModTypes.includes(modWithInfo.type)
+                selectedTechsAny.length &&
+                !selectedTechsAny.some(techId => modWithInfo.TechsAny.includes(techId))
+            ) {
+                return false;
+            }
+
+
+            if (
+                selectedTechsFC.length &&
+                !selectedTechsFC.some(techId => modWithInfo.TechsFC.includes(techId))
```

Comment:
Actually, these lines are just for enabling filtering by full clear techs - the any% tech filtering is a couple of lines up. This code isn't in use right now, but I decided to leave it in so it's easier to implement later.

Reply:
Ohh I see what you're saying. I do want to enable the flexible search later, but I agree that it could be confusing. I made some changes to enable both behaviours - please let me know what you think.